src/main/kotlin/team/mozu/dsm/application/port/in/item/DeleteItemUseCase.kt

```diff
@@ -0,0 +1,7 @@
+package team.mozu.dsm.application.port.`in`.item
+
+import java.util.*
+
+interface DeleteItemUseCase {
+    fun delete(id : UUID)
```

Comment:
```suggestion
    fun delete(id: UUID)
```
띄어쓰기를 안하고 이렇게 컨벤션을 맞추는게 좋을 것 같아요!

Reply:
2c3535a28721e4dd95e8dc9f179dd5bca9cb9211